scripts/treedb_vectordbbench_artifact.py

```
@@ -728,6 +751,650 @@ def file_identity(path: Path) -> dict[str, Any]:
+            if kind in {"heap", "allocs"}:
+                required = {
+                    "alloc_objects/count", "alloc_space/bytes", "inuse_objects/count", "inuse_space/bytes",
+                }
+                expected_default = "inuse_space/bytes" if kind == "heap" else "alloc_space/bytes"
+                return (
+                    "PeriodType: space bytes" in lines
+                    and required <= sample_types
+                    and default_types == {expected_default}
+                )
+            required = {"contentions/count", "delay/nanoseconds"}
+            return "PeriodType: contentions count" in lines and required <= sample_types
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)